src/backend/efiling-reviewer-api/src/test/java/ca/bc/gov/open/jag/efilingreviewerapi/document/validators/documentValidator/ValidateExtractedDocumentTest.java

```diff
@@ -19,6 +20,7 @@
     public static final String DOCUMENT_TYPE = "RCC";
     public static final String RESPONSE_TO_CIVIL_CLAIM = "Response to Civil Claim";
     public static final String NOT_RESPONSE_TO_CIVIL_CLAIM = "THIS IS NOT VALID";
+    public static final String RESTRICTED_DOCUMENT = "This is a temporary";
```

Comment:
same

Reply:
This is to test that it "hits" a restricted document